src/Modal/ModalLayer.tsx

```diff
@@ -63,7 +68,7 @@ function ModalLayer({
     return null;
   }
 
-  const handleClose = isBlocking ? null : onClose;
+  const handleClose = isBlocking ? undefined : onClose;
```

Comment:
This is fixing a minor type error - `handleClose` was getting passed to `<FocusOn>` via `onEscapeKey` but was sometimes `null`, whereas `onEscapeKey` expects either a function or `undefined`, not `null`.

```
Type 'null' is not assignable to type '((event: Event) => void) | undefined'.ts(2322)
types.d.ts(15, 5): The expected type comes from property 'onEscapeKey'
```